common/djangoapps/entitlements/tests/test_entitlements_api.py

```diff
@@ -0,0 +1,162 @@
+import json
+import unittest
+import uuid
+
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+from entitlements.tests.factories import CourseEntitlementFactory
+from entitlements.models import CourseEntitlement
+from student.tests.factories import CourseEnrollmentFactory, UserFactory, TEST_PASSWORD
+
+
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class EntitlementsTest(ModuleStoreTestCase):
+    """
+    Entitlements API/View Tests
+    """
+    USERNAME = 'Bob'
+    ENABLED_CACHES = ['default']
+    COURSE_RUN_ID = 'some/great/course'
+
+    def setUp(self):
+        super(EntitlementsTest, self).setUp()
+        self.user = UserFactory(is_staff=True)
+        self.client.login(username=self.user.username, password=TEST_PASSWORD)
+        self.course = CourseFactory.create(org='TestX', course='TS101', run='T1')
```

Comment:
Unless you actually care about the specific values ditch the kwargs. Also, let's remove the confusion around course vs. course run: `self.course_run = CourseFactory()`.